weave-js/src/components/PagePanelComponents/Home/Browse3/charts/types.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * types.ts
+ *
+ * This file contains all common type definitions for chart components and helpers.
+ */
+import {GridFilterModel, GridSortModel} from '@mui/x-data-grid-pro';
+
+import {WFHighLevelCallFilter} from '../pages/CallsPage/callsTableFilter';
+
+/**
+ * Structured feedback data that's easier to work with than the raw API format.
+ * Parsed once during extraction to avoid repeated parsing.
+ */
+export type ProcessedFeedback = {
+  /** Annotation values keyed by annotation type (e.g., { Quality: 8.5, Relevance: 0.8 }) */
+  annotations: {[key: string]: any};
+  /** Scorer values keyed by scorer name (e.g., { toxicity: 0.2, coherence: 0.9 }) */
+  scorers: {[key: string]: any};
+  /** Array of note strings */
+  notes: string[];
+  /** Array of reaction emojis */
+  reactions: string[];
+};
+
+export type ExtractedCallData = {
```

Comment:
this seems nearly identical to other `Call` definitions - any reason they are re-defined instead of extended?